src/CattyTests/DataModel/VariablesContainerTest.swift

```diff
@@ -96,4 +96,145 @@ final class VariablesContainerTest: XCTestCase {
         XCTAssertEqual(1, container.allLists(for: objectA)?.count)
         XCTAssertEqual(1, container.allLists(for: objectB)?.count)
     }
+
+    func testAllVariables() {
+        let objectA = SpriteObject()
+        objectA.name = "testObjectA"
+
+        let userVariable1 = UserVariable()
```

Comment:
One thing which would be great all over the project: could you add an additional optional initializer for `UserVariable` (with name and isList parameter)?